src/models/RekognitionModel.ts

```diff
@@ -12,7 +14,7 @@ export default class RekognitionModel extends BaseModel {
   }
 
   detectFaces (imageIds: string[]) {
-    return;
+    return mockResponse;
     //this.rekognition.detectFaces();
```

Comment:
Aaalmost :D I think it makes more sense to have have the logic of looping though the images in here - if you look at the arguments, we get all image strings.

I would move the mock response to the rekognition connection local :)